config/install/field.field.node.alert.field_alert_visibility_state.yml

```diff
@@ -16,7 +16,7 @@ field_name: field_alert_visibility_state
 entity_type: node
 bundle: alert
 label: 'Alert visibility state'
-description: 'Specify pages by using their paths. Enter one path per line. The ''*'' character is a wildcard. An example path is /user/* for every user page. <front> is the front page.'
+description: ''
```

Comment:
why is this removed?
Can we get Screenshot of the form attached to PR?

Reply:
![Create_Alert___YMCA_of_Central_Ohio](https://user-images.githubusercontent.com/238201/187474589-06567768-8141-4cb6-bcea-8c9fe92dce62.png)
This description is a duplicate of the one above and does not apply to the `Alert visibility state` field. This field is a simple radio and needs no description.